src/mongoose_client_api/mongoose_client_api.erl

```diff
@@ -77,6 +78,14 @@ set_resp_body_if_missing(Body, Req) ->
             cowboy_req:set_resp_body(Body, Req)
     end.
 
+maybe_report_error(StatusCode, Req, Body) when StatusCode >= 400 ->
+    ?LOG_WARNING(#{what => reply_error,
+                   stacktrace => erlang:process_info(self(), current_stacktrace),
+                   status_code => StatusCode,
+                   cowboy_req => Req, body => Body});
```

Comment:
Same issue I had before, logging the `cowboy_req` and the `body` might leak private information.

Reply:
Users would have to write log filters to filter out private info from the request.
We have cowboy_req field in docs.